config_management/roles/kubernetes-start/tasks/main.yml

```diff
@@ -11,19 +11,19 @@
     enabled: yes
 
 - name: kubeadm init on the master
-  command: 'kubeadm init --token={{ kubernetes_token }}'
```

Comment:
@errordeveloper, something like this?
```
TASK [kubernetes-start : kubeadm join on workers] ******************************
skipping: [104.154.186.116]
fatal: [104.154.216.174]: FAILED! => {"changed": true, "cmd": ["kubeadm", "join", "--token=123456.012346", "10.128.0.9"], "delta": "0:00:00.121018", "end": "2017-03-15 15:50:33.952446", "failed": true, "rc": 1, "start": "2017-03-15 15:50:33.831428", "stderr": "[discovery: Invalid value: \"123456.012346\": token [\"123456.012346\"] was not of form [\"^([a-z0-9]{6})\\\\.([a-z0-9]{16})$\"], discovery: Invalid value: \"123456.012346\": token must be of form '[a-z0-9]{6}.[a-z0-9]{16}', discovery: Invalid value: \"10.128.0.9\": missing port in address 10.128.0.9]", "stdout": "[kubeadm] WARNING: kubeadm is in beta, please do not use it for production clusters.\n[preflight] Running pre-flight checks", "stdout_lines": ["[kubeadm] WARNING: kubeadm is in beta, please do not use it for production clusters.", "[preflight] Running pre-flight checks"], "warnings": []}
```
It looks like the token gets interpreted as a double, which is then rounded up to the 6th decimal, hence not matching the regex. I'll force-stringify it.